index.js

```diff
@@ -0,0 +1,34 @@
+
+document.addEventListener('DOMContentLoaded', () => {
+
+  let ul = document.getElementById('unordered_list');
+  let submission = document.querySelector('#submission');
+  let toDo = document.querySelector('.todo');
+
+  submission.addEventListener('click', () => {
+    if (!toDo.value) {
```

Comment:
Remember, when dealing with forms, you have to pass the event to the listener and call preventDefault.
`(event) => event.preventDefault`
If not, your form will not submit properly.